### ISSUE TITLE
No nickname or name_suffix fields for legislators

### ISSUE THREAD

**Issue by konklone** [propublica/congress-api-docs#37]
In the Sunlight Congress API, legislators have both a `first_name` and a `nickname`, for when their legal name and the name they go by differs. There is also a `name_suffix`, for `Jr.` and `III` and similar.

The Pro Publica API doesn't have those fields. Could you clarify:

* It looks like name suffixes will be contained in last name fields -- is that right?
* Does the `first_name` field always contain the name the legislator goes by, even if it's not their legal first name? Is there some other rule?

**Comment by dwillis** [comment]
To clarify:

* Name suffixes are in the `last_name` fields.
* We do try and make `first_name` what the lawmaker goes by, with some exceptions: if the lawmaker uses his or her middle name as a first name, we still put the middle name in `middle_name`.

**Comment by konklone** [comment]
> * We do try and make `first_name` what the lawmaker goes by, with some exceptions: if the lawmaker uses his or her middle name as a first name, we still put the middle name in `middle_name`.

Yeah, I recall that being a common case. Do you have suggested client logic to reliably display the name the lawmaker goes by? I assume that in most `middle_name` cases, it's not the name they go by, and so you can't just always pick the `middle_name` field.

**Comment by dwillis** [comment]
We could add a `display_name` field - a similar thing appears in some responses - and standardize on that?

**Comment by konklone** [comment]
That would work! I noticed that in other responses, and that would do just fine.
